ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -504,6 +504,9 @@ gcp)
     if [ -f "${SHARED_DIR}/gcp_min_permissions.json" ]; then
       echo "$(date -u --rfc-3339=seconds) - Using the IAM service account for the minimum permissions testing on GCP..."
       export GOOGLE_CLOUD_KEYFILE_JSON="${SHARED_DIR}/gcp_min_permissions.json"
+    elif [ -f "${CLUSTER_PROFILE_DIR}/user_tags_sa.json" ]; then
```

Comment:
just like what we discussed in https://redhat-internal.slack.com/archives/CF8SMALS1/p1695816438712789?thread_ts=1695807602.653359&cid=CF8SMALS1, we need a dedicate step to copy the file from vault to generate a new sa json, then insert the step into the user_tag install provision chain. Or else, the code would be become buggy, all the gcp install on QE's  gcp cluster profile would always use this user_tag sa to run install.

Reply:
Do you mean to overwrite the ${CLUSTER_PROFILE_DIR}/gce.json with ${CLUSTER_PROFILE_DIR}/user_tags_sa.json?